components/FileUploader.js

```diff
@@ -83,46 +88,52 @@ class Filename extends Component {
   static propTypes = {
     name: PropTypes.string,
     style: PropTypes.object,
-    status: PropTypes.oneOf(['edit', 'complete', 'uploading']),
+    status: PropTypes.oneOf(['edit', 'complete', 'uploading'])
   };
 
   static defaultProps = {
     status: 'uploading',
-    style: {},
+    style: {}
   };
 
   render() {
     const { name, status, style, ...other } = this.props;
-    const tempStyle = Object.assign(style, { marginRight: '-1px' }); // temp style correction for loading component position
-    return (
-      <span>
-        {status === 'uploading'
-          ? <div className="bx--loading" style={tempStyle} {...other}>
-              <svg className="bx--loading__svg" viewBox="-42 -42 84 84">
-                <circle cx="0" cy="0" r="37.5" />
-              </svg>
-            </div>
-          : null}
-        {status === 'edit'
-          ? <Icon
-              className="bx--file-close"
-              name="close--glyph"
-              description={`Remove the file named: ${name}`}
-              style={style}
-              {...other}
-            />
-          : null}
-        {status === 'complete'
-          ? <Icon
-              className="bx--file-complete"
-              name="checkmark--glyph"
-              description={`The file named, ${name}, has been uploaded.`}
-              style={style}
-              {...other}
-            />
-          : null}
-      </span>
-    );
+
+    if (status === 'uploading') {
+      return (
+        <div
+          className="bx--loading"
+          style={Object.assign(style, { width: '1rem', height: '1rem' })}
```

Comment:
__Changes:__ Use new overriding style for Loading component to make it `1rem` size